dynamic-navbar

SaqibAjmainCareer opened the pull request:
Finally a working adjustable nav bar using only tailwind, html, and a little javascript.

Still minor QOL issues like not having the search field fit the page in mobile view, but this can be overlooked for now.

SaqibAjmain committed:
dynamic-navbar

Finally a working adjustable nav bar using only tailwind, html, and a little javascript.

Still minor QOL issues like not having the search field fit the page in mobile view, but this can be overlooked for now.

SaqibAjmainCareer committed:
Merge branch 'main' into bugfix/adaptive-nav-bar

SaqibAjmain committed:
added some styling to the results page

simple styling to the results page

Carechiga reviewed (commented):
Make sure when you're added the href attribute your path begins "./" not just "/"

SaqibAjmain committed:
styling

additional styling to the results page, there still needs to be work done, but this is a big push forward towards the final product.

The only thing that needs to be changed now is the alignment of the cards on the page, they need to be center, and the standardization of the card size.

SaqibAjmain committed:
change

one change

SaqibAjmain committed:
good catch

good catch from Cristino

SaqibAjmain committed:
change

change
